Source/Scene/PostProcessBloomStage.js

```diff
@@ -0,0 +1,144 @@
+define([
+        '../Core/Check',
+        '../Core/defineProperties',
+        '../Core/destroyObject',
+        '../Shaders/PostProcessFilters/BloomComposite',
+        '../Shaders/PostProcessFilters/ContrastBias',
+        './PostProcess',
+        './PostProcessBlurStage',
+        './PostProcessComposite'
+    ], function(
+        Check,
+        defineProperties,
+        destroyObject,
+        BloomComposite,
+        ContrastBias,
+        PostProcess,
+        PostProcessBlurStage,
+        PostProcessComposite) {
+    'use strict';
+
+    /**
+     * Post process stage for bloom. Implements {@link PostProcess}.
+     *
+     * @alias PostProcessBloomStage
+     * @constructor
+     *
+     * @private
```

Comment:
Why is this, blur, etc. private?  Because they are exposed through the scene's post processes?

Reply:
Blur isn't private anymore. Bloom, AO, and FXAA are private because they're already in the collection.